various-db-engines-it/db-it/pom.xml

```diff
@@ -18,6 +18,7 @@
     <module>mysql-it</module>
     <module>postgresql-it</module>
     <module>oracle-it</module>
+    <module>mongodb-it</module>
```

Comment:
@akoniar defining another module here is fine, but isn't a `pom.xml` in that module missing?